specification/api/components/schemas/LetterItem.yml

```diff
@@ -0,0 +1,17 @@
+properties:
+  type:
+    type: string
+    const: Letter
+  id:
+    type: string
+    examples:
+      - 2WL5eYSWGzCHlGmzNxuqVusPxDg
+  attributes:
+    type: object
+    properties:
+      status:
+        $ref: "./LetterStatus.yml"
+      specificationId:
+        type: string
+        examples:
+          - 2WL5eYSWGzCHlGmzNxuqVusPxDg
```

Comment:
I think there's a confusion here between `LetterStatusItem` and `LetterItem` - they should be the same schema I think?

I know we need to have the discussion around metadata so we can leave a decision on anything else for later.